docs/content/connectors/oceanbase-cdc.md

```diff
@@ -0,0 +1,492 @@
+        <tr>
+            <td>tenant_name</td>
+            <td>STRING NOT NULL</td>
+            <td>Name of the tenant that contains the row.</td>
+        </tr>
+        <tr>
+            <td>database_name</td>
+            <td>STRING NOT NULL</td>
+            <td>Name of the database that contains the row.</td>
+        </tr>
+        <tr>
+            <td>table_name</td>
+            <td>STRING NOT NULL</td>
+            <td>Name of the table that contains the row.</td>
+        </tr>
+        <tr>
+            <td>op_ts</td>
+            <td>TIMESTAMP_LTZ(3) NOT NULL</td>
+            <td>It indicates the time that the change was made in the database. <br>
+                If the record is read from snapshot of the table instead of the change stream, the value is always 0.</td>
+        </tr>
+    </tbody>
+</table>
+
+The extended CREATE TABLE example demonstrates the syntax for exposing these metadata fields:
+
+```sql
+CREATE TABLE products (
+    tenant_name STRING METADATA FROM 'tenant_name' VIRTUAL,
+    db_name STRING METADATA FROM 'database_name' VIRTUAL,
+    table_name STRING METADATA  FROM 'table_name' VIRTUAL,
+    operation_ts TIMESTAMP_LTZ(3) METADATA FROM 'op_ts' VIRTUAL,
+    order_id INT,
+    order_date TIMESTAMP(0),
+    customer_name STRING,
+    price DECIMAL(10, 5),
+    product_id INT,
+    order_status BOOLEAN,
+    PRIMARY KEY(order_id) NOT ENFORCED
+) WITH (
+   'connector' = 'oceanbase-cdc',
+   'scan.startup.mode' = 'initial',
+   'username' = 'user@test_tenant',
+   'password' = 'pswd',
+   'tenant-name' = 'test_tenant',
+   'database-name' = 'test_db',
+   'table-name' = 'orders',
+   'hostname' = '127.0.0.1',
+   'port' = '2881',
+   'rootserver-list' = '127.0.0.1:2882:2881',
+   'logproxy.host' = '127.0.0.1',
+   'logproxy.port' = '2983');
+```
+
+Features
+--------
+
+### At-Least-Once Processing
+
+The OceanBase CDC connector is a Flink Source connector which will read database snapshot first and then continues to read change events with **at-least-once processing**.
+
+### Startup Reading Position
+
+The config option `scan.startup.mode` specifies the startup mode for OceanBase CDC consumer. The valid enumerations are:
+
+- `initial`: Performs an initial snapshot on the monitored table upon first startup, and continue to read the latest commit log.
+- `latest`: Never to perform snapshot on the monitored table upon first startup and just read the latest commit log since the connector is started.
+- `timestamp`: Never to perform snapshot on the monitored table upon first startup and just read the commit log from the given `scan.startup.timestamp`.
+
+### Consume Commit Log
+
+The OceanBase CDC Connector using [oblogclient](https://github.com/oceanbase/oblogclient) to consume commit log from OceanBase LogProxy.
+
+### DataStream Source
+
+The OceanBase CDC connector can also be a DataStream source. You can create a SourceFunction as the following shows:
+
+```java
+public class OceanBaseSourceExample {
```

Comment:
Adds the imports used by OceanBaseSourceExample.

Reply:
Done.